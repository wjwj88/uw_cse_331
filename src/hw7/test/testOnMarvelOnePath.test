# Load the Marvel dataset and search for an existing path. hw7
# Find one path between two nodes

LoadGraph marvel marvel.tsv

FindPath marvel OLD_SKULL HAWK