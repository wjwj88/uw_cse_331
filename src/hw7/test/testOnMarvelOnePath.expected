# Load the Marvel dataset and search for an existing path. hw7
# Find one path between two nodes

loaded graph marvel

path from OLD SKULL to HAWK:
OLD SKULL to HAWK with weight 0.077
total cost: 0.077